app/views/partners/children/_form.html.erb

```diff
@@ -17,10 +17,10 @@
               <%= form.text_field :last_name, class: "form-control" %>
 
               <%= form.label :item_needed, "Diaper/Item Used" %>
-              <%= form.select :item_needed_diaperid, @formatted_requestable_items,
-                              {include_blank: 'Select an item'},
-                              {class: 'form-control'} %>
-              <br>
+              <span>(select multiple by holding `ctrl`)</span>
```

Comment:
@cielf thoughts about this? Typically we have an interface similar to the line item where we give them an "add new" button. Is there going to be confusion / difficulty using the interface if it requires a select-click?

Reply:
Just saw this - will try to take a look tomorrow.

Reply:
My take:  

I like the aesthetics of the select-many approach better,  but I think having to use 'ctrl'  does have the potential to be confusing .  One issue is that the key that is acting as 'ctrl' is actually 'command' on Macs -- using 'control', which is what you'd *think* would work,  doesn't.   (I tried it out)  

The add new pattern is relatively clunky, aesthetically, but since having more than 2 items per child is relatively rare, I think it's workable.